.github/workflows/unit-tests.yml

```diff
@@ -22,10 +22,11 @@ defaults:
 
 jobs:
   ubuntu:
-    runs-on: ubuntu-22.04
+    runs-on: ${{ matrix.platform }}
     timeout-minutes: 90
     strategy:
       matrix:
+        platform: [ubuntu-22.04, ubuntu-24.04-arm]
```

Comment:
I think the  job `name:` and concurrency `group:` will need to account for `matrix.platform` now